cmd/remove.go

```diff
@@ -72,7 +73,7 @@ func (o *RemoveOptions) Complete(f Factory, args []string) (err error) {
 		return err
 	}
 	if o.Refs, err = GetCurrentRefSelect(f, args, -1, nil); err != nil {
-		return err
+		return lib.NewError(lib.ErrBadArgs, "please specify a dataset path or name you would like to remove from your qri node")
```

Comment:
Near as I could tell, `o.Validate` was never getting called, so I just moved its logic here. Not sure if that’s kosher, though. In another spot with a similar setup, I called `o.Validate()` after `o.Complete()`.